### PR TITLE
Remove use of global variables

### PR DESCRIPTION
**Pull request by JSchoenbachler**

### CODE REVIEW

**Comment by jakejh** [comment]
1. Under the theme of being more careful, please fix the name of the PR.
2. For columns in data.tables operations or variables in foreach loops, I'd advocate for putting all NULL assignments in a given function on one line.

```r
group2Now = i = NULL
```

3. For columns used in ggplots, we should follow the guidelines in the [vignette](https://ggplot2.tidyverse.org/articles/ggplot2-in-packages.html) that Dora mentioned, which means adding `#' @importFrom rlang .data`, using `aes(x = .data$col)` instead of `aes(x = col)`, and likely adding rlang explicitly to the imports section of the Description file.

**Comment by JSchoenbachler** [comment]
1. Fixed
2. That works for me. I don't think anyone in the lab really uses that notation otherwise so it would stick out that way.
3. Okay, I can make that change!

**Comment by JSchoenbachler** [comment]
@jakejh @ehoutland I made the suggested changes. All seems to still be well!

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# deltaccd 0.2.4
```
Comment (jakejh) [inline NEWS.md]: fix version number and rebuild pkgdown site.

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+* Replaced `globalVariable` call to get around `devtools::check` warning with null variable assignment and using the rlang .data variable for ggplot2.
```
Comment (jakejh) [inline NEWS.md]: - function is called `globalVariables`.
- refer to R CMD check instead of just `devtools::check`.

File: R/deltaccd_plot.R [hunk 3]
```diff
@@ -31,8 +33,9 @@ calcColorLimits = function(
 
 
 plotHeatmapSimple = function(ggObj, cLims) {
+  gene1 = gene2 = rho = NULL
```
Comment (jakejh) [inline R/deltaccd_plot.R]: using `.data` means NULL assignment is unnecessary

**Review by jakejh — CHANGES_REQUESTED** [review]

**Comment by JSchoenbachler** [comment]
These issues are resolved!

File: NEWS.md (hunk 1 above)
Comment (JSchoenbachler) [reply NEWS.md]: Fixed!

File: NEWS.md (hunk 2 above)
Comment (JSchoenbachler) [reply NEWS.md]: Fixed!

File: R/deltaccd_plot.R (hunk 3 above)
Comment (JSchoenbachler) [reply R/deltaccd_plot.R]: Fixed!

**Comment by JSchoenbachler** [comment]
@jakejh re-requesting review!

**Review by jakejh — APPROVED** [review]
Looks ok to me.

**Review by ehoutland — APPROVED** [review]
I think it looks good.

**Pull request merged**